README.md

```diff
@@ -77,6 +77,21 @@ resolver.resolve(did)
   .catch(error => 'failed');
  ```
 
+#### Using as driver
+You can also use this project as a did resolver driver. This project is developed based on the guidelines of [Decentralized Identity](https://github.com/decentralized-identity/did-resolver/tree/master#implementing-a-did-method)
+You can call it simply with calling `getResolver()`:
+```typescript
+const { getResolver, Resolver } = require('../src/resolver/Resolver');
+
+const did = 'did:btcr:xz35-jznz-q6mr-7q6';
+const didResolutionResult1 = await getResolver()
+  .resolve(did);
+const didResolutionResult2 = await getResolver()
+  .resolve(did,
```

Comment:
This looks like a variation of the same problem. Again, I believe the options should come from the getResolver method, so that method needs to get an optional opts argument. In the end the resolver is not accessible directly for the end user as it will be wrapped by the DIF resolver. Look at the below example:

`
ethrResolver = ethr.getResolver()
webResolver = web.getResolver()

//If you are using multiple methods you need to flatten them into one object
const resolver = new Resolver({
  ...ethrResolver,
  ...webResolver,
})
`

From that point on I will be using resolver to do the resolution and not ethrResolver or webResolver directly.
`
resolver.resolve('did:uport:2nQtiQG6Cgm1GYTBaaKAgr76uY7iSexUkqX/some/path#fragment=123')
`
So I simply cannot pas in the options at the moment of resolution, I have to configure them when creating our uni-resolver object. The only way to do that is to add an optional options argument in the getResolver() call if you ask me